rocket_twin/systems/physics/dynamics.py

```diff
@@ -0,0 +1,74 @@
+from cosapp.base import System
+
+
+class Dynamics(System):
+    """Dynamics of a physical system.
+
+    Inputs
+    ------
+    forces [N]: float,
+        total force in each component of the system
+    weights [kg]: float,
+        total weight of each component of the system
+    centers[m]: float,
+        center of gravity of each component of the system
+
+    Outputs
+    ------
+    force [N]: float,
+        total force
+    weight [kg]: float,
+        total weight
+    a [m/s**2] : float,
+        acceleration
+    """
+
+    def setup(self, forces=None, weights=None, centers=None):
+        if forces is None:
+            forces = []
+        if weights is None:
+            weights = []
+        if centers is None:
+            centers = []
+
+        self.add_property("forces", forces)
+        self.add_property("weights", weights)
+        self.add_property("centers", centers)
+
+        # Checks if rocket is mid-flight or still in station
+        self.add_inward_modevar(
+            "flight", False, desc="Whether the rocket is flying or not", unit=""
+        )
```

Comment:
you should not have Rocket info in generic Dynamic system

Reply:
Check